app/views/about/index.html.slim

```diff
@@ -0,0 +1 @@
+h1 About Page
```

Comment:
Prefer to move all static strings to I18n localizations from the very beginning in order to easily moderate them without making changes to views but just editing locales